esiclient/v1/switch.py

```diff
@@ -124,6 +124,32 @@ def take_action(self, parsed_args):
         return ["Port", "VLANs"], data
 
 
+class ListSwitches(command.Lister):
+    """List Switches """
+
+    log = logging.getLogger(__name__ + ".ListSwitches")
+
+    def get_parser(self, prog_name):
+        parser = super(ListSwitches, self).get_parser(prog_name)
+        return parser
+
+    def take_action(self, parsed_args):
+        self.log.debug("take_action(%s)", parsed_args)
+
+        ironic_client = self.app.client_manager.baremetal
+
+        ports = list((port for port in ironic_client.port.list(detail=True)))
+
+        data = []
+        for port in ports:
+            switch = port.local_link_connection.get('switch_info')
+            switch_id = port.local_link_connection.get('switch_id')
+
+            if [switch, switch_id] not in data:
+                data.append([switch, switch_id])
+        return ["Switch Name", "Switch Id"], data
```

Comment:
`Id` should be `ID` (both letters capitalized)

Reply:
Done!